stix_shifter_modules/carbonblack/stix_translation/data_mapping.py

```diff
@@ -10,8 +10,8 @@ def __init__(self, options):
         assert "process" in self.map_data and "binary" in self.map_data
 
     def fetch_mapping(self):
-        process_mapping = self._fetch_mapping_file("process_from_stix_map.json")
-        binary_mapping = self._fetch_mapping_file("binary_from_stix_map.json")
+        process_mapping = self._fetch_mapping_file("process_from_stix_map_part.json")
```

Comment:
wondering why it is necessary to change the file name? basically you added `_part`

Reply:
carbonblack is the only connector that combines two mapping into the default no name dialect. Dialect discovery logic is based on the filenames, so for the carbonblack it has to find no mapping files for the dialects so only the default dialect is represented for it.